products/error_tracking/backend/prompts.py

```diff
@@ -0,0 +1,227 @@
+from datetime import datetime
```

Comment:
Tested this out locally. Didn't have any errors generating updates but there was certainly some unexpected results

1. First it did a raw search of email, then used the property for two very similar questions
<img width="1079" height="566" alt="Image" src="https://github.com/user-attachments/assets/fe2409f2-16e4-4790-bfe7-0c41c0893b55" />

2. did not know how to apply the internal user filter and instead added a property I'm pretty sure doesn't exist
<img width="1266" height="181" alt="Image" src="https://github.com/user-attachments/assets/453d3ba4-9014-43bc-99c3-d5bccbc43281" />